### ISSUE TITLE
How to access sovereign admin panel

### ISSUE THREAD

**Issue by rong-00gle** [sovereign/sovereign#821]
My installation has been completed with playbook.

Now i wanna create my email users but i could find admin panel for that.
Please  add some more documents how to use after installation.

**Comment by neuhaus** [comment]
There is no admin panel included with sovereign. I'm using a password protected instance of VEA https://github.com/neuhaus/vea myself to configure email accounts.

I agree we need to improve the documentation. Pull requests are welcome!
We also have a wiki at https://github.com/sovereign/sovereign/wiki

**Comment by amishtechie** [comment]
How would some get involved with documenting the project?

**Comment by Titan-C** [comment]
Just do it. Open the wiki and write your experience on how you got something working. Your future self will thank you, when you need to do it again.

**Comment by neuhaus** [comment]
You can either add to the wiki on github or edit one of the text files and submit a pull request. Your work is highly appreciated!

**Comment by jdmoore0883** [comment]
What about just using Webmin as an "admin ui"? It might not be able to administer some of the services like wallabag and znc et al., but it it may be able to provide a useful gui for some of the more common server services like users and apache and ufw/iptables and such.

Is there any specific reason preventing someone from using webmin (and/or including it in the nice-to-haves maybe)?

**Comment by neuhaus** [comment]
Not really, we are using the standard db scheme for postfix so email should work with most frontends (i'm not familiar with them).
